src/SFA.DAS.ApprenticeCommitments.Web/Api/RegistrationsService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using SFA.DAS.ApprenticeCommitments.Web.Api.Models;
+using System;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeCommitments.Web.Api
+{
+    public class RegistrationsService
+    {
+        private readonly ApiClient _client;
+
+        public RegistrationsService(ApiClient client) => _client = client;
+
+        public Task<Registration> GetRegistration(Guid id) => _client.GetRegistration(id);
```

Comment:
The APIM implementation of /registration/{guid} returns a 404 if not existing and a 200 if it exists (with the model). Will our code fall over when it receives a 404 (I'm guessing it will be seen by the library as a invalid response)

Reply:
We can tell Restease to give us the status in cases where it makes sense: .ncrunchsolution